### ISSUE TITLE
No speed or GPS data on Dride video and none detected by the player (iOS 6.4.0/FW 1.3)

### ISSUE THREAD

**Issue by MyDefender** [dride/dride-app-issues#65]
So, I'm using the HQBKING Player to play my Dride video. There is a GPS_LOG folder on the MicroSD Card and it has '.raw' files in it which seem to correspond to '.mp4' files in the Movie folder. When I play a Dride video there is no speed or GPS data detected by the HQBKING Player and none recorded at the bottom on the video either. 
![2020-01-04 (1)](https://user-images.githubusercontent.com/56696655/71769844-db6afb80-2f1d-11ea-9553-4d3c7945823f.png)

**Comment by BilliTheKid** [comment]
I've noticed with the latest app update its hit or miss with the GPS coordinates again. Hopefully @saoron will look into it.

**Comment by MyDefender** [comment]
I’ve never had any GPS data show up on my videos no matter how long the journey. @saoron keeps saying that it takes a few minutes to get a GOS signal, but I still don’t get anything after half an hour!

**Comment by BilliTheKid** [comment]
> I’ve never had any GPS data show up on my videos no matter how long the journey. @saoron keeps saying that it takes a few minutes to get a GOS signal, but I still don’t get anything after half an hour!

Yes issues with GPS. I have reported it to @saoron No update yet.

**Comment by BilliTheKid** [comment]
![Screenshot_20200108-192254](https://user-images.githubusercontent.com/56843472/72008734-6c0e4800-324c-11ea-94e9-6db3fc7714ad.png)
![Screenshot_20200108-192242](https://user-images.githubusercontent.com/56843472/72008735-6ca680-324c-11ea-88d5-433e5045236d.png)
![Screenshot_20200108-192231](https://user-images.githubusercontent.com/56843472/72008736-6ca6de80-324c-11ea-973f-ea2285dc31e3.png)
@saoron

**Comment by BilliTheKid** [comment]
![Screenshot_20200108-192254](https://user-images.githubusercontent.com/56843472/72008792-80524500-324c-11ea-9bea-c8219314e19f.png)

**Comment by BilliTheKid** [comment]
![Screenshot_20200108-192242](https://user-images.githubusercontent.com/56843472/72008816-8a744380-324c-11ea-84fa-3297fba33fa2.png)

**Comment by BilliTheKid** [comment]
@MyDefender Try deleting all videos and then check again in 5 minutes. After I done this GPS coordinates appeared again. It maybe a bug with storage if many vids but I'm unsure

**Comment by BilliTheKid** [comment]
This issue seems to be resolved. Please close thread. @MyDefender @saoron

**Comment by MyDefender** [comment]
It’s not resolved for me. Still no GPS data after deleting all videos @saoron @BilliTheKid

**Comment by BilliTheKid** [comment]
> It’s not resolved for me. Still no GPS data after deleting all videos @saoron @BilliTheKid

Oh really? No issues in my end. Strange.

**Comment by MyDefender** [comment]
@saoron @BilliTheKid I have GPS data on my videos for the first time ever, after finally being able to update to V1.4 of the FW.  I had already updated to FW 1.4 from GitHub which resulted in not being able to connect to the app. I then followed the instructions in https://dride.io/forum/-fix-for-firmware-14-____Y6OFQaNmbmlgtTsXwdOx, download V.1.4.1, put that on the SD card but it would not install - probably because it is named 'FW96663A-1.4.1.bin' and not 'FW96663A.bin'. I ended up installing V1 of the FW and was then able to update to 1.4 via the app. I now seem to have GPS data on the recording for the first time ever! Let's hope that is not a passing phase :)
